layers/+lang/clojure/packages.el

```diff
@@ -83,6 +83,7 @@
             "eb" 'cider-eval-buffer
             "ee" 'cider-eval-last-sexp
             "ef" 'cider-eval-defun-at-point
+            "ev" 'cider-eval-sexp-at-point
```

Comment:
Sort this alphabetically, like the other key bindings, that makes it easier to find which keys are used/available, without having to look through the whole list.

And add an entry for it in the clojure readme > evaluation section:
https://github.com/syl20bnr/spacemacs/tree/develop/layers/+lang/clojure#evaluation

Reply:
I actually tried to keep it sorted by the CIDER function so all the "eval" related ones are grouped together. Is that OK or is it better to sort by the command? 

I'll squash commits and add the docs as you note though.